vllm/core/block/interfaces.py

```diff
@@ -186,6 +186,10 @@ def get_num_blocks_touched(self,
                                num_lookahead_slots: int = 0) -> int:
         pass
 
+    def prefix_cache_hit_rate(self) -> float:
+        """Prefix cache hit rate. -1 means not supported or disabled."""
+        return -1
```

Comment:
nit: better to keep it abstract so this file doesn't have any implementation logic (trying to keep them as strictly interfaces, not interface+impl)